__tests__/__snapshots__/URLNode.test.js.snap

```diff
@@ -2,13 +2,13 @@
 
```

Comment:
Should we just get rid of Jest? Feels like it just forces you to run a friggin snapshot update and commit new snapshot everytime we make something.

Reply:
I would not be opposed to removing Jest. I can't say that I have done much more than re-run the snapshots in order for my changes to pass. I also don't necessarily feel like the Jest comparison is particularly useful because it is not super human friendly